preconditions/iotics_test.sh

```diff
@@ -1,29 +1,25 @@
 #!/bin/bash
 
-IOTICSPACE_DOMAIN="https://ganymede.iotics.space"
-RESOLVER_DOMAIN="did.prd.iotics.com"
+SPACE_NAME="ganymede"
```

Comment:
You should be able to make this executable so it can be run right after checkout.